domain/src/main/java/com/kdjj/domain/di/UseCaseModule.kt

```diff
@@ -0,0 +1,38 @@
+package com.kdjj.domain.di
+
+import com.kdjj.domain.model.Recipe
+import com.kdjj.domain.model.RecipeType
+import com.kdjj.domain.request.*
+import com.kdjj.domain.usecase.*
+import com.kdjj.domain.usecase.FetchLocalLatestRecipeListUseCase
+import com.kdjj.domain.usecase.FetchRemoteLatestRecipeListUseCase
+import com.kdjj.domain.usecase.SaveRecipeUseCase
+import dagger.Binds
+import dagger.Module
+
+@Module
+abstract class UseCaseModule {
+    @Binds
```

Comment:
여기도 공백하나 추가하는게 좋아보입니다!
```suggestion
abstract class UseCaseModule {

    @Binds
```

Reply:
추가했습니다!